src/components/existing-request/existing-request-display.vue

```diff
@@ -424,7 +424,16 @@ export default class ExistingRequestDisplay extends Mixins(
 
   /** True if the Register button should be shown. */
   get showRegisterButton (): boolean {
-    return this.isFirm(this.nr)
+    return this.isFirm(this.nr) &&
+           this.nr.request_action_cd !== RequestCode.CHG &&
+           (NrState.APPROVED === this.nr.state ||
+            !this.isConcentRequired)
+  }
```

Comment:
Instead of negative logic ("not CHG"), I think it would be better to positively compare with the allowable request action code (NEW, I think).

Reply:
There are total 10 request_action_cds. I think it is better to keep Not CHG.

Reply:
"Not CHG" means this might be true for 8 invalid request action codes, but we only care about the one valid code (NEW).